GooglePlayInstant/Editor/QuickDeploy/DialogHelper.cs

```diff
@@ -102,14 +102,18 @@ public static string SaveFilePanelInProject(string title, string defaultPath, st
         /// </summary>
         private static string AbsoluteToAssetsRelativePath(string absolutePath)
```

Comment:
Do you mind adding unit tests? You could do this by having this method take Application.dataPath as a parameter.